src/main/kotlin/no/nav/amt/aktivitetskort/service/HendelseService.kt

```diff
@@ -42,7 +42,12 @@ class HendelseService(
 	fun deltakerlisteHendelse(id: UUID, deltakerliste: DeltakerlisteDto?) {
 		if (deltakerliste == null) return
 
-		when (val result = deltakerlisteRepository.upsert(deltakerliste.toModel())) {
+		if (!deltakerliste.tiltakstype.erStottet()) return
+
+		val arrangor = arrangorRepository.get(deltakerliste.virksomhetsnummer)
+			?: throw NoSuchElementException("Fant ikke arrangør med organisasjonsnummer ${deltakerliste.virksomhetsnummer}")
```

Comment:
For å unngå feil kan du hente fra amt-arrangør hvis arrangøren mangler. Alternativt kan du ha en retry og satse på at amt-tiltak eller bff vil trigge en oppretting av arrangøren og at den skrives til topic :) Hvis det tar litt tid kan det hende du må kjøre lytterne på arrangør og deltakerliste som to ulike kafkaconsumers da, for hvis de blir stående lenge og feile kan det hende at de blokkerer hele lytteren så arrangøren ikke kommer inn (jeg lærte dette the hard way i bff-en 😅)

Reply:
Det er. litt uheldig om kakfalytteren blokkerer alle topics hvis en feiler ja, her hadde vi egentlig bare lagt opp til at ting må komme i riktig rekkefølge, men at hvis det kommer en deltakerliste først, så må det også komme en arrangør, for den var allerede opprettet i amt-tiltak men det må vi kanskje revurdere litt.